src/vs/workbench/contrib/positronIPyWidgets/test/browser/positronIPyWidgetsService.test.ts

```diff
@@ -225,18 +307,20 @@ suite('Positron - IPyWidgetsInstance constructor', () => {
 	let logService: ILogService;
 	let session: TestLanguageRuntimeSession;
 	let messaging: TestIPyWidgetsWebviewMessaging;
+	let notebookService: INotebookService;
 
 	setup(async () => {
 		logService = new NullLogger() as unknown as ILogService;
 		session = disposables.add(new TestLanguageRuntimeSession());
 		messaging = disposables.add(new TestIPyWidgetsWebviewMessaging());
+		notebookService = new TestNotebookService() as unknown as INotebookService;
```

Comment:
You shouldn't actually need the `as unknown` here and removing will potentially save headaches down the line?